### ISSUE TITLE
Bugfix: When resizing the window the map moves down a few centimetres (pan) -- magically fixed somehow

### ISSUE THREAD

**Issue by offeringofpie** [lemiffe/ai-streetview#3]

**Comment by offeringofpie** [comment]
@lemiffe will this help? https://leafletjs.com/reference.html#resizeevent
or https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver
[or something like](https://leafletjs.com/reference.html#map-invalidatesize):
```
map.on('map-container-resize', () => {
   map.invalidateSize();
});
```

**Comment by lemiffe** [comment]
yeah that was already there in the default implementation of Leaflet for Svelte, but it doesn't work... ever since I made the container larger for some reason it jumps down when resizing - weird one

**Comment by offeringofpie** [comment]
It's also a fringe issue, no?

The only situation I see that we have to seriously account for window resizing is when rotating a phone screen.
